doc/source/changes/version_0_24.rst.inc

```diff
@@ -57,3 +57,9 @@ Fixes
 * fixed the array() method on excel.Sheet returning float labels when int labels are expected.
 
 * fixed getting float data instead of int when converting an Excel Sheet or Range to an larray or numpy array.
+
+* fixed crash of Session.save method when it contains a 0D array.
+  0D arrays are skiped when saving a session (closes issue:`291`).
+
+* fixed crash of Session.save method when trying to save a session in an Excel that does not exist.
+  A new Excel file is created if it does not exist yet (closes issue:`313`).
```

Comment:
isn't crash a too strong word for something which failed with a relatively nice error message? I would write something like:

    fixed Session.save and Session.to_excel failing to create new Excel files (it only worked if the file already existed). Closes ...